app/javascript/dashboard/routes/dashboard/settings/inbox/components/CampaignTable.vue

```diff
@@ -0,0 +1,272 @@
+            </div>
+          ),
+        },
+
+        {
+          field: 'message',
+          key: 'message',
+          title: this.$t('CAMPAIGN.LIST.TABLE_HEADER.MESSAGE'),
+          align: 'left',
+          width: 300,
+          renderBodyCell: ({ row }) => {
+            return (
+              <div class="text-truncate">
+                <span title={row.message}>{row.message}</span>
+              </div>
+            );
+          },
+        },
+        {
+          field: 'enabled',
+          key: 'enabled',
+          title: this.$t('CAMPAIGN.LIST.TABLE_HEADER.STATUS'),
+          align: 'left',
+          renderBodyCell: ({ row }) => {
+            return (
+              <div>
+                <span class={`status status__${row.enabled}`}>
+                  {row.enabled
+                    ? this.$t('CAMPAIGN.LIST.STATUS.ENABLED')
+                    : this.$t('CAMPAIGN.LIST.STATUS.DISABLED')}
+                </span>
+              </div>
+            );
+          },
+        },
+
+        {
+          field: 'sender',
+          key: 'sender',
+          title: this.$t('CAMPAIGN.LIST.TABLE_HEADER.SENDER'),
+          align: 'left',
+          renderBodyCell: ({ row }) => {
+            if (row.sender)
+              return (
+                <div class="row--user-block">
+                  <Thumbnail
+                    src={row.sender.thumbnail}
+                    size="20px"
+                    username={row.sender.name}
+                    status={row.sender.availability_status}
+                  />
+                  <div>
+                    <h6 class="sub-block-title user-name text-truncate">
+                      {row.sender.name}
+                    </h6>
+                  </div>
+                </div>
+              );
+            return '---';
+          },
+        },
+        {
+          field: 'url',
+          key: 'url',
+          title: this.$t('CAMPAIGN.LIST.TABLE_HEADER.URL'),
+          align: 'left',
+          renderBodyCell: ({ row }) => (
+            <div class="text-truncate">
+              <a
+                target="_blank"
+                rel="noopener noreferrer nofollow"
+                href={row.url}
+                title={row.url}
+              >
+                {row.url}
+              </a>
+            </div>
+          ),
+        },
+        {
+          field: 'timeOnPage',
+          key: 'timeOnPage',
+          title: this.$t('CAMPAIGN.LIST.TABLE_HEADER.TIME_ON_PAGE'),
+          align: 'left',
+        },
+
+        {
+          field: 'buttons',
+          key: 'buttons',
+          title: '',
+          align: 'left',
+          renderBodyCell: () => (
+            <div class="button-wrapper">
+              <WootSubmitButton
+                button-text={this.$t('CAMPAIGN.LIST.BUTTONS.EDIT')}
+                icon-class="ion-edit"
+                button-class="link hollow grey-btn"
+                click="openEditPopup(label)"
```

Comment:
Shouldn't this be the following?

```
<WootSubmitButton
  :button-text={this.$t('CAMPAIGN.LIST.BUTTONS.EDIT')}
  icon-class="ion-edit"
  button-class="link hollow grey-btn"
  @click="openEditPopup(label)"
/>

Reply:
VeTable needs JSX syntax for column templates if we using imports directly I think.

Reply:
@nithindavid You are right